docs/source/quick_start.rst

```diff
@@ -3,23 +3,432 @@ Quick start
+
+    $ bash 2_sync_command.sh
+
+Then remove ``lochness.json`` for security ::
+
+    $ rm lochness.json
+
+
+
+4. Edit ``config.yml``::
+
+    $ vim config.yml
+
+
+Edit AWS s3 bucket name and root directory ::
+
+    AWS_BUCKET_NAME: pronet-test
+    AWS_BUCKET_ROOT: TEST_PHOENIX_ROOT_PRONET
+
+
+Edit ``base`` field for Box structure ::
+
+    box:
+        PronetLA:
+            base: ProNET/PronetLA
+            delete_on_success: False
+            file_patterns:
+                actigraphy:
+                       - vendor: Activinsights
+                         product: GENEActiv
+                         data_dir: PronetLA_Actigraphy
+                         pattern: '*.*'
+                eeg:
+                       - product: eeg
+                         data_dir: PronetLA_EEG
+                         pattern: '*.*'
+                interviews:
+                       - product: open
+                         data_dir: PronetLA_Interviews/OPEN
+                         out_dir: open
+                         pattern: '*.*'
+                       - product: psychs
+                         data_dir: PronetLA_Interviews/PSYCHS
+                         out_dir: psychs
+                         pattern: '*.*'
+                       - product: transcripts
+                         data_dir: PronetLA_Interviews/transcripts/Approved
+                         out_dir: transcripts
+                         pattern: '*.*'
+
+
+
+Run ``sync.py``
+---------------
+
+Execute ``sync.py`` script to have lochness to continuously sync data ::
+
+    sync.py -c /data/pronet/data_sync_pronet/config.yml \
+        --studies PronetLA PronetOR PronetBI PronetNL PronetNC PronetSD \
+                  PronetCA PronetYA PronetSF PronetPA PronetSI PronetPI \
+                  PronetNN PronetIR PronetTE PronetGA PronetWU PronetHA \
+                  PronetMT PronetKC PronetPV PronetMA PronetCM PronetMU \
+                  PronetSH PronetSL \
+        --source redcap upenn box xnat mindlamp \
+        --lochness_sync_send --s3 \
+        --debug --continuous
+
+
+This will run lochness sync function for each site (``studies``) for all
+measures (``source``). It will upload newly downloaded data to the s3 bucket
+after each data sweep for all sources. Then, this ``sync.py`` function will
+execute these functions again after ``poll_interval`` stated in the
+``config.yml``.
+
+
+``lochness_create_template.py`` creates a template bash script that could be
+used. ::
+
+    bash 2_sync_command.sh
+
+
+
+
+Example PHOENIX-BIDS structure
+------------------------------
+
+U24 uses ``PHOENIX-BIDS`` structure, which is slightly different from the
+``PHOENIX`` structure. ``PHOENIX-BIDS`` was used to have more similarity to the
+``BIDS`` structure, while maintaining ``protected`` vs ``general`` and ``raw``
+vs ``processed`` concept of the ``PHOENIX``.
+
+
+**Summary of the structure**
+
+``<protected>/<study>/<processed>/<subject>/<datatypes>`` ::
```

Comment:
Did you mean to remove this line? I am wondering because the following tree shows `raw` primarily.